Urban Synergy Unleashed/common/on_actions/MoG_USU_on_actions.txt

```diff
@@ -36,10 +50,8 @@ mog_urban_on_action = {
 
 
 ## MONTHLIES
-on_monthly_pulse_country = {
+on_monthly_pulse_state = {
```

Comment:
I find it runs pretty smoothly, so I'm not sure staggering it by state is really necessary. What's the thinking here?